tlslite/keyexchange.py

```diff
@@ -268,6 +267,12 @@ def processClientKeyExchange(self, clientKeyExchange):
             raise TLSLocalAlert(AlertDescription.illegal_parameter,
                                 "Invalid dh_Yc value")
 
+        """if dh_Yc % dh_p == 0:
+            for result in self._sendError(AlertDescription.illegal_parameter,
+                    "Suspicious dh_Yc value"):
+                yield result
+            assert(False)  # Just to ensure we don't fall through somehow"""
```

Comment:
the check is correct, but it should raise an exception directly, as this function is not a generator, `TLSIllegalParameterException` for example